### PR TITLE
Integrated Danger to post diffs generated on Travis

### PR DESCRIPTION
**Pull request by RomanTysiachnik**
## Failure example
Here is an example when Snapshot test fail and we want to have a look on diff images without running these tests again on a personal machine:

- Install aws cli and set up aws access keys from Travis environmental variables.
- Snapshot tests fail;
-  We jump to the `after_failure` step, where:
   * All images are removed from s3 bucket for current branch if there were any;
   * All new images are deployed;
   * The list of images on `bucket/Failures/branch_name` is saved in a `.json` file as an array of strings;
   * Then launches a Dangerfile with a script to post failures for each failed test case, with one shown image from it and with all others, hidden under the spoiler.

### Preview of the comment:

screen shot 2018-02-27 at 1 36 11 pm

In this example, we can see a test target and a test case that has failed, one image with diffs and a Details block with all other images.

## Success example
- If Travis exits with 0, all images for this branch on s3 will be deleted;
- Danger launches, doesn't find a json file with keys and removes a comment from PR,

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,10 +5,26 @@ xcode_workspace: PlayerControls
 before_install:
 - brew update
 - brew install sourcery
-- bundle install
+- bundle update
```
[the remark this answers is not in the document]
Comment (RomanTysiachnik) [reply .travis.yml]: @AndriiMoskvin should we use `install` to download gems from `.lock` file? If yes, then I'll fix it.

[the remark this answers is not in the document]
Comment (AndriiMoskvin) [reply .travis.yml]: @BogdanBilonog Yes, exactly.

File: Gemfile [hunk 2]
```diff
@@ -1,3 +1,7 @@
 source "https://rubygems.org"
 
+git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }
```
Comment (AndriiMoskvin) [inline Gemfile]: Why do we need this?

Comment (RomanTysiachnik) [reply Gemfile]: It is actually generated by Bundle. We get these gems from RubyGems, not from GitHub, so we don't need that.

**Comment by AndriiMoskvin** [comment]
@RomanTysiachnik rebase and merge please

**Pull request merged**